### PR TITLE
[CLIENT-7502] Emit samples even when MOS is `null`.

### PR DESCRIPTION
**Pull request by mhuynh5757**
**Contributing to Twilio**

> All third party contributors acknowledge that any contributions they provide will be made under the same open source license that the open source project is provided under.

- [x] I acknowledge that all my contributions will be made under the project's license.

### JIRA link(s):

- [CLIENT-7502](https://issues.corp.twilio.com/browse/CLIENT-7502)

### Description

This PR introduces a change to Preflight Test such that samples emitted by the connection are also emitted by the test even when MOS is `null`.

### Before review
* [x] Updated CHANGELOG.md if necessary
* [x] Added unit tests if necessary
* [x] Updated affected documentation
* [x] Verified locally with `npm test`
* [x] Manually sanity tested running locally
* [x] Ready for review

### Before merge
* [ ] Got one or more +1s
* [ ] Squashed erroneous commits if necessary
* [ ] Re-tested if necessary

### CODE REVIEW

File: PREFLIGHT_TWIML.md [hunk 1]
```diff
@@ -31,7 +31,7 @@ Using the [TwiML Bin](https://www.twilio.com/console/twiml-bins) page, let's cre
 
 <Response>
   <Say>Record a message in 3, 2, 1</Say>
-  <Record maxLength="3" action="https://my-record-twiml-url"></Record>
+  <Record maxLength="5" action="https://my-record-twiml-url"></Record>
```
Comment (charliesantos) [inline PREFLIGHT_TWIML.md:34]: Testing this more, it doesn't seem to solve the problem and it only feels weird because we now have a longer pause. I think we need to lengthen the TwiML playback, same approach we did with network test tool. That is, let's put this back to 3 seconds and update the `TwiML App -Playback` template to the following:

```xml
<?xml version="1.0" encoding="UTF-8"?>
<Response>
  <Say>You said:</Say>
  <Play loop="1">{{RecordingUrl}}</Play>
  <Say>Now waiting fifteen seconds to gather audio performance metrics.</Say>
  <Pause length="15"/>
  <Say>Hanging up now.</Say>
</Response>
```

Comment (charliesantos) [reply PREFLIGHT_TWIML.md:34]: Actually, waiting 15 seconds might trigger constant audio level warnings. Let's keep the maxlength of 5 seconds in your PR, and update the `TwiML App - Playback` to this instead. What do you think?
```xml
<?xml version="1.0" encoding="UTF-8"?>
<Response>
  <Say>You said:</Say>
  <Play loop="1">{{RecordingUrl}}</Play>
  <Say>Now waiting a few seconds to gather audio performance metrics.</Say>
  <Pause length="3"/>
  <Say>Hanging up now.</Say>
</Response>
```
